### ISSUE TITLE
iOS VoiceOver ignores the GroupedList row body elements

### ISSUE THREAD

**Issue by sergul** [microsoft/fluentui#13104]
### Environment Information

- **Package version(s)**: 7.105.10
- **Browser and OS versions**: Safari, iOS - v. 13.4.1

### Describe the issue:

iOS VoiceOver doesn't read / ignores the GroupedList body elements when navigating via
left -> right finger swipes.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/sergulo/pen/KKdoxjq

#### Actual behavior:

iOS Screen reader (VoiceOver) doesn't focus and announce the GroupedList body element when navigating via left->right finger swipe.
Instead VoiceOver jumps out of GroupedList item when all the item header elements are announced and lands onto next list item header.

#### Expected behavior:
iOS Screen reader (VoiceOver) should focus and announce the GroupedList item body elements when navigating via left->right finger swipe.
It should not jump out of GroupedList item and land onto next list item header but should continue reading the first item body elements when navigating via left->right finger swipe.

**Comment by aneeshack4** [comment]
@sergul I just tried to repro this on the latest iOS and Safari and couldn't - all of the elements of the GroupedList were receiving proper focus in the right order and being announced correctly including the body element. I also just tried this with VoiceOver on Mac to show you the captions since I can't do that on iOS and I had no problem there either: 
![image](https://user-images.githubusercontent.com/4161791/81716685-459d2b00-943f-11ea-8256-2745eb56480a.png)

**Comment by sergul** [comment]
@aneeshack4 thanks for the feedback and I see it works in the example you pointed out. However the way we use GroupedList renders header i.e. "onRenderHeader: this._onRenderHeader"

So if you try an example where GroupedList renders a header then you should be able repro this issue.
![image](https://user-images.githubusercontent.com/2910631/81719327-c8b78480-9431-11ea-8757-d65d809596a3.png)

**Comment by sergul** [comment]
Closing this bug as I found a workaround shown in snapshot below:
![image](https://user-images.githubusercontent.com/2910631/81728526-72514280-943f-11ea-9a56-c2ae0eac95ec.png)
